### PR TITLE
Insert the shader name in the source.

### PR DESCRIPTION
**Pull request by nical**
This makes life easier when looking at apitrace recordings.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/1821)

### CODE REVIEW

File: webrender/src/device.rs [hunk 1]
```diff
@@ -183,6 +183,11 @@ pub fn build_shader_strings(
     vs_source.push_str(gl_version_string);
     fs_source.push_str(gl_version_string);
 
+    // Insert the shader name to make debugging easier.
+    let name_string = format!("// {}\n", base_filename);
```
Comment (kvark) [inline webrender/src/device.rs]: perhaps, it would make sense to do in debug only?

Comment (nical) [reply webrender/src/device.rs]: I use apitrace with release builds too and I don't think this adds any noticeable overhead so I lean towards leaving it in release builds as well unless you feel strongly about it.

**Comment by kvark** [comment]
Okey
@bors-servo r+

**Pull request merged**